### DISCUSSION TITLE
Welcome to xmake-vscode Discussions!

### DISCUSSION THREAD

**Discussion by waruqi** [xmake-io/xmake-vscode#116] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rdabra** [comment]
I work in a multi-project project and could not independently build a specific subproject  ("xmake -P ."). I think it would be great to add "-P ." functionalities to Xmake: configure, Xmake: build and  Xmake: run.
